secure_message/common/alerts.py

```diff
@@ -27,18 +27,18 @@ def send(email, reference):
                                  json=notification)
 
         if response.status_code != 201:
```

Comment:
Gateway returns `201` for a success?  As in temporary redirect?

Reply:
201 means created